### DISCUSSION TITLE
generating genotypes for multiple generations

### DISCUSSION THREAD

**Discussion by MoeenNehzati** [tskit-dev/msprime#2059] [Q&A]
Hi,
I'm in need of genetic data encompassing multiple generations that kind of mimic European ancestry. I've been able to generate a population with my desired traits using msprime but it's just one generation. Does msprime support any way of outputting parents or offsprings of this population?
Thanks!

**Comment by petrelharp** [comment]
Hi, @MoeenNehzati! Let's see - it depends on exactly what you want to do, but you could do something like: switch to [discrete-time Wright-Fisher](https://tskit.dev/msprime/docs/stable/ancestry.html#discrete-time-wright-fisher) for the last few generations, and [take some samples from previous generations](https://tskit.dev/msprime/docs/stable/ancestry.html#specifying-samples) by specifying some [SampleSets](https://tskit.dev/msprime/docs/stable/api.html#msprime.SampleSet). If you want to specifically sample a given number of parent-offspring pairs, though, then you'd want to do something with [simulating in a fixed pedigree](https://tskit.dev/msprime/docs/stable/ancestry.html#sec-ancestry-models-fixed-pedigree).
